ucs/clientlist.cpp

```diff
@@ -793,7 +793,10 @@ void Clientlist::ProcessOPMailCommand(Client *c, std::string command_string, boo
 	case CommandJoin:
 		if (!command_directed) {
 			//Append saved channels to params
-			parameters = parameters + ", " + database.CurrentPlayerChannels(c->GetName());
+			const auto saved_channels = database.CurrentPlayerChannels(c->GetName());
+			if (!saved_channels.empty()) {
+				parameters += fmt::format(", {}", Strings::Join(saved_channels, ", "));
```

Comment:
Did you test this - don't think this will do what you think it will

Reply:
Seems to work fine in my testing